package.json

```diff
@@ -43,7 +43,7 @@
     "test": "npm run lint && npm run unit && npm run integration-api",
     "unit": "node scripts/build/cli npmCiModules && npm run unit-webapp && npm run unit-admin && npm run unit-shared-lib && npm run unit-api && npm run unit-sentinel",
     "unit-admin": "node ./scripts/ci/run-karma.js",
-    "unit-api": "UNIT_TEST_ENV=1 nyc --nycrcPath='api/nyc.config.js' mocha 'api/tests/mocha/**/*.js'",
+    "unit-api": "UNIT_TEST_ENV=1 nyc --nycrcPath='api/nyc.config.js' mocha 'api/tests/mocha/services/sso-login.spec.js' 'api/tests/mocha/**/*.js'",
```

Comment:
This is more weirdness caused by `openid-client` being an ES Module. :disappointed: As [described here](https://github.com/medic/cht-core/pull/9833#discussion_r2049603474), there is some weird behavior when combining `require`ing an ESM + `nyc` + mocha tests _with file globbing_.  Explicitly targeting `sso-login.spec.js` works around this issue (and all the unit tests are still only run once).